src/main/java/neqsim/thermodynamicoperations/flashops/SolidFlash12.java

```diff
@@ -392,9 +392,8 @@ public double solvebeta1() {
       iter++;
       solidCandidate = system.getPhase(0).getComponents()[solidIndex].getz();
       for (int i = 0; i < system.getNumberOfPhases(); i++) {
-        solidCandidate -=
-            system.getPhases()[3].getComponent(solidIndex).fugcoef(system.getPhases()[3])
-                / system.getPhase(i).getComponent(solidIndex).getFugacityCoefficient();
+        solidCandidate -= system.getPhase(3).getComponent(solidIndex).fugcoef(system.getPhase(3))
```

Comment:
In this case I beleive we have to used system.getPhases()[3] - as the solid phase is always number three (and getPhase(3) can point to a different phase.